src/py/_pyodide/_core_docs.py

```diff
@@ -1238,6 +1238,39 @@ def to_js(
 
     Examples
     --------
+    >>> from js import Object, Map, Array
+    >>> js_object = pyodide.ffi.to_js({'age': 20, 'name': 'john'}, dict_converter=Object.fromEntries)
+    >>> js_object.age == 20
+    True
+    >>> js_object.name == 'john'
+    True
+    >>> js_object.toString()
+    [object Object]
+    >>> js_object.hasOwnProperty("age")
+    True
+    >>> js_object.hasOwnProperty("height")
+    False
+
+    >>> js_object = pyodide.ffi.to_js({'age': 20, 'name': 'john'}, dict_converter=Map.new)
+    >>> js_object.keys(), js_object.values()
+    KeysView([object Map]) ValuesView([object Map])
+    >>> [(k, v) for k, v in zip(js_object.keys(), js_object.values())]
+    [('age', 20), ('name', 'john')]
```

Comment:
Since this is the default behavior, maybe put this example first and leave off `dict_converter`? Also, what about having a `>>> js_object` after we assign to it?

Reply:
Removed the param dict_converter=Map.new as it is the default behavior